Drv/LinuxSpiDriver/LinuxSpiDriverComponentImpl.cpp

```diff
@@ -97,9 +98,27 @@ namespace Drv {
 
         // Configure:
         /*
-         * SPI Mode 0
+         * SPI Mode 0, 1, 2, 3
          */
-        U8 mode = SPI_MODE_0; // Mode 0 (CPOL = 0, CPHA = 0)
+
+        U8 mode; // Mode Select (CPOL = 0/1, CPHA = 0/1)
+        switch(spiMode) {
+            case SpiMode::SPI_MODE_0:
+                mode = SPI_MODE_0;
+                break;
+            case SpiMode::SPI_MODE_1:
+                mode = SPI_MODE_1;
+                break;
+            case SpiMode::SPI_MODE_2:
+                mode = SPI_MODE_2;
```

Comment:
Just noticed! Missing `break` statements.

Reply:
Thanks! Just added the missing statements